driver/src/main/java/org/neo4j/driver/internal/logging/JULogger.java

```diff
@@ -26,21 +26,22 @@
 public class JULogger implements Logger
 {
     private final java.util.logging.Logger delegate;
-    private final boolean debugEnabled;
 
     public JULogger( String name )
     {
         delegate = java.util.logging.Logger.getLogger( name );
-        debugEnabled = delegate.isLoggable( Level.FINE );
```

Comment:
If we keep this, then we only have the chance to enable and disable debug mode during initialization. I somehow added a FINEST level so this check seems not so meaningful at all.